paymentsheet/src/main/java/com/stripe/android/paymentsheet/IntentConfirmationHandler.kt

```diff
@@ -367,7 +371,9 @@ internal class IntentConfirmationHandler(
                 PaymentConfirmationResult.Failed(
                     cause = IllegalStateException(message),
                     message = R.string.stripe_something_went_wrong.resolvableString,
-                    type = PaymentConfirmationErrorType.MerchantIntegration,
+                    type = PaymentConfirmationErrorType
+                        .MerchantIntegration
+                        .GooglePayCurrencyCodeNotProvidedInSetupMode,
```

Comment:
We weren't logging merchant integration errors before but so I figured we could also log the Google Pay one as well.